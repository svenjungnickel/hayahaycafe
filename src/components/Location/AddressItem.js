import React from 'react';
import PropTypes from 'prop-types';

const AddressItem = ({ item }) => (
    <>
        {item}
        <br />
    </>
);

AddressItem.propTypes = {
    item: PropTypes.string.isRequired,
};

export default AddressItem;
