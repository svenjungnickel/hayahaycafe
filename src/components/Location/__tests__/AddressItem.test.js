import React from 'react';
import renderer from 'react-test-renderer';
import AddressItem from '../AddressItem';

describe('AddressItem', () => {
    it('renders correctly', () => {
        const item = 'item';

        const component = <AddressItem item={item} />;
        const tree = renderer.create(component).toJSON();

        expect(tree).toMatchSnapshot();
    });
});
