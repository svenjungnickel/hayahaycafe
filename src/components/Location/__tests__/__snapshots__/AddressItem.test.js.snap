// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddressItem renders correctly 1`] = `
[
  "item",
  <br />,
]
`;
